jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/WebSocketLocalEndpoint.java

```diff
@@ -28,25 +28,24 @@
  */
 public interface WebSocketLocalEndpoint
 {
-    Logger getLog();
+    Logger getLog(); // TODO why?
```

Comment:
This make the logs appear as named from the users endpoint.

Reply:
Need convincing, but minor point.